sharepoint/sharepoint-ps/sharepoint-online/Add-SPOContentSecurityPolicy.md

```diff
@@ -15,28 +15,27 @@ ms.reviewer:
 
 ## SYNOPSIS
 
-Adds an entry to the **Content Security Policy** configuration.
+Adds a source to the **Content Security Policy** configuration.
 
 ## SYNTAX
 
 ### Default
 
 ```powershell
-Add-SPOContentSecurityPolicy [-Url] <String> [-Directive] <String> 
+Add-SPOContentSecurityPolicy [-Source] <String>
 ```
 
 ## DESCRIPTION
 
-Adds an entry to the **Content Security Policy** configuration. 
-The url in each entry will be added to the corresponding directive during construction of the Content-Security-Policy header.
-In multi-geo environments **Content Security Policy** entries are unique to each geo.
-Entries with a "*" directive will be applied to all directives.
+Adds a source to the **Content Security Policy** configuration. 
+The source will be added to the `script-src` directive during construction of the `Content-Security-Policy` header.
```

Comment:
Should we add a pointer to the CSP documentation for what is supported in the source parameter? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/Sources#sources

Reply:
There's no precedent I can see for linking to the mdn docs, but I do think it's a good idea. Will get Whitney's thoughts on if it's appropriate.